### DISCUSSION TITLE
How to sync all opened tabs on Kiwi@android?

### DISCUSSION THREAD

**Discussion by somehow22** [floccusaddon/floccus#1483] [Q&A] [answered]
newbie to floccus, so not sure is it me or a general usage problem. I don't use bookmarks anymore(save them to pinboard only) and usually opened 20 or more tabs with browser on my android phone and I would need to organize them with laptop some days later. I've been using Edge so no problem with syncing, but after switching to kiwi(for extension capabilities), it's a thing. 

I just installed floccus 5.0.4 from github release and use selfhosted WebDAV for syncing. Syncing is fine, but it only "uploaded" one or two active tabs, all inactive tabs were ignored. How to force floccus to sync all tabs opened? thanks!

**Comment by marcelklehr** [comment, marked as answer]
This is a known issue: https://github.com/floccusaddon/floccus/issues/1032
